src/query-builder/transformer/RawSqlResultsToEntityTransformer.ts

```diff
@@ -146,6 +147,36 @@ export class RawSqlResultsToEntityTransformer {
         return hasData;
     }
 
+    protected transformComputedColumns(rawResults: any[], alias: Alias, entity: ObjectLiteral): boolean {
+        let hasData = false;
+
+        this.expressionMap.computedSelects.forEach(computedSelect => {
```

Comment:
Why the `forEach` over `for..of`?

Reply:
Personally I prefer `for...of` too but I continue with the way of developing in the document. For example, you can see the `transformColumns` method with a `forEach`:

```let hasData = false;
        metadata.columns.forEach(column => {
                      ...... // CODE
        });```